example/main.tf

```diff
@@ -23,3 +28,18 @@ module "distribution" {
   vpc_id     = var.vpc_id
   subnet_ids = var.subnet_ids
 }
+
+module "s3-replicator" {
```

Comment:
How is this going to be released?

Reply:
Definitely not 100% of the time. I think I need to make this opt-in, which means adding some documentation about "how to opt-in."